### PR TITLE
{bio}[foss/2018b] CITE-seq-Count v1.4.3 w/ Python 3.6.6

### PR DESCRIPTION
**Pull request by timeu**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/python-Levenshtein/python-Levenshtein-0.12.0-foss-2018b-Python-3.6.6.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['033a11de5e3d19ea25c9302d11224e1a1898fe5abd23c61c7c360c25195e3eb1']
+
+dependencies = [
+    ('Python', '3.6.6'),
+]
+
```
Comment (verdurin) [inline easybuild/easyconfigs/p/python-Levenshtein/python-Levenshtein-0.12.0-foss-2018b-Python-3.6.6.eb]: @timeu Any reason not to `use pip` with this one?

Comment (timeu) [reply easybuild/easyconfigs/p/python-Levenshtein/python-Levenshtein-0.12.0-foss-2018b-Python-3.6.6.eb]: No reason good. 
I think I just forgot it to add it. I added them

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 2 out of 4 (3 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.7.1908, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/6f273f4026c6152ff17ef969fbbb4395 for a full test report.

**Comment by verdurin** [comment]
@timeu Here's the specific error:
```
== 2019-11-11 14:10:22,855 build_log.py:164 ERROR EasyBuild crashed with an error (at easybuild/base/exceptions.py:124 in __init__): Sanity check failed: no (non-empty) directory found at 'lib/python3.6/site-packages/python_Levenshtein-0.12.0-py3.6-linux-x86_64.egg' in /eb/maint/software/python-Levenshtein/0.12.0-foss-2018b-Python-3.6.6 (at easybuild/framework/easyblock.py:2585 in _sanity_check_step)
== 2019-11-11 14:10:22,855 easyblock.py:3082 WARNING build failed (first 300 chars): Sanity check failed: no (non-empty) directory found at 'lib/python3.6/site-packages/python_Levenshtein-0.12.0-py3.6-linux-x86_64.egg' in /eb/maint/software/python-Levenshtein/0.12.0-foss-2018b-Python-3.6.6
== 2019-11-11 14:10:22,855 easyblock.py:294 INFO Closing log for application name python-Levenshtein version 0.12.0
```

File: easybuild/easyconfigs/p/python-Levenshtein/python-Levenshtein-0.12.0-foss-2018b-Python-3.6.6.eb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+download_dep_fail = True
+use_pip = True
+
+options = {'modulename': 'Levenshtein'}
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/python_Levenshtein-%(version)s-py%(pyshortver)s-linux-x86_64.egg'],
```
Comment (verdurin) [inline easybuild/easyconfigs/p/python-Levenshtein/python-Levenshtein-0.12.0-foss-2018b-Python-3.6.6.eb]: As mentioned, the test build finds a problem with this.

**Comment by smoors** [comment]
Test report by @smoors
**FAILED**
Build succeeded for 2 out of 3 (3 easyconfigs in this PR)
node251.hydra.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2650 v4 @ 2.20GHz, Python 2.7.5
See https://gist.github.com/beedec0f1d828a3e361317b8fcf3bebd for a full test report.

**Comment by timeu** [comment]
@smoors: Thanks for the test.
 I forgot to add the pytest dependency which I fixed in the last commit. Can you give it another try ?

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node217.hydra.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 2.7.5
See https://gist.github.com/849d5e37e0c8b7a8cc24c40c0d5f2a6a for a full test report.

**Comment by smoors** [comment]
@timeu the checks failed because PSI4 and ProjectQ depend on pytest-3.8.2, so you should use the same version for CITE-seq-Count.

**Comment by timeu** [comment]
@smoors : Hmm I see but in there is only a pytest-3.8.2-foss-2017b-Python-3.6.3.eb (2017b) toolchain. 
There is a pytest-4.3.0-foss-2018b-Python-3.6.6.eb easyconfig. 
So this means I would have to create a pytest-3.8.2-foss-2018b-Python-3.6.3.eb version ?

**Comment by smoors** [comment]
@timeu yes, unfortunately (but using Python-3.6.6). the alternative is to add pytest and all its deps to the extensions list.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node251.hydra.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2650 v4 @ 2.20GHz, Python 2.7.5
See https://gist.github.com/da3293f6d8e643a827e83eb9775d0f5e for a full test report.

**Comment by smoors** [comment]
Going in, thanks @timeu!

**Pull request merged**